app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -434,7 +425,6 @@ class HomeFragment : Fragment() {
         }
         homeViewModel.layoutManagerState =
             sessionControlView.view.layoutManager?.onSaveInstanceState()
-        homeViewModel.motionLayoutProgress = homeLayout?.progress ?: 0F
```

Comment:
The way to fix this would be to target the AppBar in the onResume most probably, since when re-entering that fragment, we'd have to collapse it again manually. I think we can probably do that by just by doing that with our `List<Tab>` since with x # of tabs, we'd have to collapse it I think?  I'll look more at it to make sure I'm thinking about this right.

Reply:
@mcomella  I get the same behavior in the previous version 😕  Should I fix it ? Its a quick one liner

Reply:
@mcomella this should now be fix! if you want to try , I just pushed the change that would fix this 😄